lib/services/marketplace-plugins-service.ts

```diff
@@ -11,7 +11,7 @@ export class MarketplacePluginsService implements ICordovaPluginsService {
 		private $projectConstants: Project.IProjectConstants) { }
 
 	public getAvailablePlugins(): IFuture<any> {
-		return this.$server.cordova.getMarketplacePluginsData();
+		return this.$server.cordova.getMarketplacePluginsData(this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova);
```

Comment:
maybe we could use 
```TypeScript
this.$project.projectData.Framework
```
This way we'll not change the code in the future, when we have nativescript plugins :)